tests/framework/db/mysql/QueryBuilderTest.php

```
@@ -29,6 +31,245 @@ final class QueryBuilderTest extends BaseQueryBuilder
     protected $driverName = 'mysql';
     protected static string $driverNameStatic = 'mysql';
 
+    public function testBuildOrderByAndLimitWithOffsetAndLimit(): void
+    {
+        $qb = $this->getConnection(false)->getQueryBuilder();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $qb. Configured minimum length is 3. (undefined)

(ShortVariable)